### PR TITLE
Implement as.tbl_json.data.frame for #27

### LINKED ISSUE #27: Turn data.frames with a JSON column into a tbl_json object
From David, a data scientist at socure.me:

*"My question is whether it can be used on a data frame with a number of regular columns and one json column. Ideally, I'd like to expand just the json column into its constituent columns, and then bind them back with the rest of the dataframe. Is this possible?"*

One way to implement this would be through a `as.tbl_json.data.frame` function that takes a mandatory second argument which identifies the column of the source `data.frame` that contains the JSON data.

This is useful for:
* JSON that was added to an existing data.frame (perhaps via multiple API calls)
* Tables exported from relational databases that have JSON columns (e.g., Postgres)

This should also support `dplyr::data_frame` and `dplyr::tbl.df` source objects.

### PR DESCRIPTION
**Pull request by jeremystan**
Closes #27

### CODE REVIEW

File: R/tbl_json.r [hunk 1]
```diff
@@ -60,6 +60,27 @@ as.tbl_json.character <- function(x, ...) {
...
+#' @param json.column the name of the JSON column of data in x, if x is a data.frame
+#' @rdname tbl_json
+#' @export
+as.tbl_json.data.frame <- function(x, json.column, ...) {
+  
+  assert_that(is.character(json.column))
+  assert_that(json.column %in% names(x))
+
+  # Parse the json
+  json <- lapply(x[[json.column]], fromJSON, simplifyVector = FALSE)
```
Comment (adgaudio) [inline R/tbl_json.r]: is it clear which fromJSON will get applied?  If the user uses both jsonlite and rjson, does it matter which one gets used?   rjson::fromJSON might be a good replacement

Comment (adgaudio) [inline R/tbl_json.r]: And to that end, we should make all of tidyJSON's 3rd party library calls explicit.

**Comment by adgaudio** [comment]
after you address my comment, +1

File: R/tbl_json.r (hunk 1 above)
Comment (jeremystan) [inline R/tbl_json.r]: It's actually `jsonlite::fromJSON` that is used. `jsonlite` is imported in `R/tidyjson-package.r`, and R package namespaces will guarantee `tidyjson` will always use the `jsonlite` version of `fromJSON`.

**Pull request merged**